BUILD.bazel

```diff
@@ -4,12 +4,44 @@
 
 load("@io_kubernetes_build//defs:run_in_workspace.bzl", "workspace_binary")
 load("@build_bazel_rules_nodejs//:defs.bzl", "nodejs_binary")
+load("@io_bazel_rules_go//go:def.bzl", "nogo")
 
 workspace_binary(
     name = "go",
     cmd = "@go_sdk//:bin/go",
 )
 
+nogo(
+    name = "nogo_vet",
```

Comment:
I don't see this getting used anywhere

Reply:
It's referenced in WORKSPACE in `go_register_toolchains`:

```python
go_register_toolchains(
    go_version = "1.11.5",
    nogo = "@//:nogo_vet",
)
```